INSTALLING.rst

```diff
@@ -1,73 +1,71 @@
-----------------------
+Anaconda
+--------
 
 For all Windows, macOS and Linux platforms, it is recommended to use Anaconda_
 to install the latest release of pycalphad. Anaconda is a scientific Python
-distribution by Anaconda, Inc. It provides good support for various
-scientific packages and otherwise challenging to install packages.
+distribution by Anaconda, Inc. It provides good support for various scientific
+packages and otherwise challenging to install packages.
 
 To install pycalphad from Anaconda
 
 1. Download and install Anaconda_
-2. From the Anaconda Prompt (Windows) or a terminal emulator (macOS and Linux) run ``conda install -c pycalphad -c msys2 -c conda-forge pycalphad``
-
-PyPI
-----
+2. From the Anaconda Prompt (Windows) or a terminal emulator (macOS and Linux) run ``conda install -c pycalphad -c conda-forge pycalphad``
 
-Release versions of pycalphad are available on PyPI. As of pycalphad 0.6,
-the `Ipopt`_ library is used for numerical optimization and must be installed
-separately in order to install pycalphad from PyPI. Instructions for downloading
-and installing Ipopt are found at https://www.coin-or.org/Ipopt/documentation/node10.html.
 
-NumPy, SciPy and Cython are all *build* requirements of pycalphad and must be
-installed before you install pycalphad.
-
-To install pycalphad from PyPI using pip:
+Development Versions (Advanced Users)
+-------------------------------------
 
-1. Download and install `Ipopt`_
-2. Run the ``pip install numpy scipy cython`` command in a terminal emulator
-3. Run the ``pip install pycalphad`` command to install pycalphad
-4. (Optional) run ``pip install jupyter`` to install the Jupyter Notebook application
+Installing a development version of pycalphad will allow the latest changes and
+features in pycalphad to be used by running the bleeding edge version of the
+code, even if those changes have not been released in a tagged version to PyPI
+or Anaconda.
 
+Running the development versions allows pycalphad to be run directly from the
+modifiable source code, so you can add new features and contribute them back to
+the project.
 
-Development Versions (Advanced Users)
--------------------------------------
+Installing
+~~~~~~~~~~
 
-To install a development version of pycalphad, you can use either an Anaconda or
-vanilla Python distribution.
+To install the latest development version of pycalphad, run from the Anaconda
+Prompt (Windows) or a terminal emulator (macOS or Linux):
 
+1. Install pycalphad and the conda-forge C and C++ compilers ``conda create -c pycalphad -c conda-forge c-compiler cxx-compiler pycalphad``
+#. Remove the installed pycalphad package so the development version can be installed ``conda remove --force pycalphad``
+#. Get the pycalphad source ``git clone https://github.com/pycalphad/pycalphad.git pycalphad/`` (or download from https://github.com/pycalphad/pycalphad)
+#. Go to the top level directory of the package ``cd pycalphad``
+#. Run ``pip install -e .``
```

Comment:
Don't you need to activate the environment?

Reply:
It was intended to be `conda install`, rather than `conda create`. I'm comfortable simplifying the workflow to install into the base environment, leaving the users who understand and need the conda environment model to use/discover it on their own.